src/protocol/chromium-bidi.ts

```diff
@@ -104,7 +119,14 @@ export type CommandResponse =
   | WebDriverBidi.CommandResponse
   | Cdp.CommandResponse;
 
-export type Event = WebDriverBidi.Event | Cdp.Event;
+export type BluetoothEvent = {
+  type: 'event';
+} & (
+  | WebDriverBidiBluetooth.Bluetooth.RequestDevicePromptOpened
```

Comment:
Shouldn't `WebDriverBidiBluetooth.Bluetooth.RequestDevicePromptOpened` be `WebDriverBidi.Extensible` as well?

Reply:
Not sure, but it does not seem to cause any issues currently.